lib/diagnostic.js

```diff
@@ -60,7 +60,7 @@ let deathStar = {
 // Question 4: Write the code that would set the `deathStar`'s `fire` method
 // in the context of `alderaan` but does not invoke the method,
 // so that we could call answer4() // Alderaan
-let answer4 = null;  // replace null with your response
+let answer4 = new deathStar('Alderaan');  // replace null with your response
```

Comment:
This has a similar problem to the previous answer, but instead of calling the method we just want to bind it.

```javascript
deathStar.fire.bind(alderaan);
```